suggested changes to welcome blog post (#1)
mathtechorg/mathtechorg.github.io: _pages/about.md

title: "About"
---

> MathTech.org is a news feed for showcasing and advocating for technology

Author: ```suggestion
MathTech.org is a news feed for showcasing and advocating for open-source technology
```

> MathTech.org is a news feed for showcasing new developments in technology

Author: ```suggestion
MathTech.org is a news feed for showcasing and advocating for technology
```
